### PR TITLE
Fix vencord settings not working when below activity or default is choosed 

### PR DESCRIPTION
**Pull request by happyendermangit**
image

### CODE REVIEW

File: src/plugins/_core/settings.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default definePlugin({
                     case "bottom": return /\{section:(\i\.\i)\.CUSTOM,\s*element:.+?}/;
                     case "aboveActivity":
                     default:
-                        return /\{section:(\i\.\i)\.HEADER,\s*label:(\i)\.\i\.Messages\.ACTIVITY_SETTINGS/;
+                        return /\{section:(\i\.\i)\.HEADER,\s*label:(\i)\.\i\.Messages\.ACTIVITY_SETTINGS,.+?\}/;
```
Comment (Vendicated) [inline src/plugins/_core/settings.tsx:55]: what's the difference?

**Comment by Vendicated** [comment]
works on my machine

File: src/plugins/_core/settings.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ import gitHash from "~git-hash";
 export default definePlugin({
     name: "Settings",
     description: "Adds Settings UI and debug info",
-    authors: [Devs.Ven, Devs.Megu],
+    authors: [Devs.Ven, Devs.Megu, Devs.HAPPY_ENDERMAN],
```
Comment (Nuckyz) [inline src/plugins/_core/settings.tsx:29]:
```suggestion
    authors: [Devs.Ven, Devs.Megu],
```
❌❌❌❌❌❌

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/plugins/_core/settings.tsx (hunk 2 above)
Comment (happyendermangit) [reply src/plugins/_core/settings.tsx:29]: what?

File: src/plugins/_core/settings.tsx (hunk 1 above)
Comment (Vendicated) [reply src/plugins/_core/settings.tsx:55]: that makes no sense, it doesn't break the regex

you're just adding addiitional matchers that don't change anything about functionality. this change makes no sense

**Comment by happyendermangit** [comment]
uhm, why close the pr when people are having issues with vencord settings not showing when below activity is choosed

**Comment by happyendermangit** [comment]
image

**Comment by Cynosphere** [comment]
> uhm, why close the pr when people are having issues with vencord settings not showing when below activity is choosed

Because it is a low quality PR where all it looks like is you're trying to cobble things together just to get le funny contributor badge

**Comment by happyendermangit** [comment]
huh, so your telling me i posted a low quality PR to get the contributor badge? your so funny. like bro i just solved an problem i dont even care to have a badge

**Comment by Vendicated** [comment]
> uhm, why close the pr when people are having issues with vencord settings not showing when below activity is choosed

because this pr does absolutely nothing.

settings were already fixed via https://github.com/Vendicated/Vencord/commit/89c82e2cd173b26b0eee938022afad5684e174cd

this pr just makes the regex longer, it doesn't even change anything. There is exactly one match anyway and the outcome is exactly the same

**Comment by happyendermangit** [comment]
btw how will users update for this fix that you implemented?

**Comment by EdVraz** [comment]
> btw how will users update for this fix that you implemented?

https://github.com/Vencord/Installer
and
image

use your brain.

**Comment by happyendermangit** [comment]
thanks.

**Comment by happyendermangit** [comment]
you can lock this pr chat if you want

**Pull request closed without merging**